tests/e2e/provisioning/pkg/client/kyma/kyma_client.go

```diff
@@ -0,0 +1,69 @@
+package kyma
```

Comment:
IMO this code should be moved to runtime or runtime here. Having runtime and Kyma does not make any sense

Reply:
I removed a kyma package and moved that struct after renaming it to `dashboard_checker` to the runtime package.